usaspending_api/api_contracts/contracts/v2/autocomplete/psc.md

```diff
@@ -10,9 +10,15 @@ This endpoint is used by the Advanced Search page.
 This route sends a request to the backend to retrieve product or service (PSC) codes and their descriptions based on a search string. This may be the 4-character PSC code or a description string.
 
 + Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object"
+            }
+
     + Attributes (object)
         + `limit` (optional, number)
-            + Default: 10
```

Comment:
Was this meant to be removed? Just wondering cause the autocomplete naics keeps it.

Reply:
Yes, the updates to API Contracts are a WIP. I intended to make more changes but the diff in this PR is already large.

The direction we're moving is to remove "example/default" values from the schema definition and instead store example values in the `Body` section. There are a few exceptions to this,
required parameters being the biggest example, but overall it should clean up the contract and add additional benefit